operatorcourier/parse_manifests.py

```diff
@@ -0,0 +1,82 @@
+import os
```

Comment:
Can we name this file a noun instead of a verb? Like `manifest_helper` or even `manifest_parser` ?

Reply:
@kevinrizza Done.